modules/microshift-data-cleaning-ovn-data.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * microshift_troubleshooting/microshift-cleanup-data.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-data-cleaning-ovn-data_{context}"]
+= Cleaning the OVN data
+
+Cleanup the OVN data by running the following script. This script is useful to reset OVN settings after modifying network configurations. This script performs the following cleanup actions:
```

Comment:
```suggestion
Cleanup the OVN-K data by running the following script. This script is useful to reset OVN-K settings after modifying network configurations. 

* When to you run the script, you perform the following cleanup actions:
```

Reply:
It's not the script. It's the specific option that should be stressed here.